### PR TITLE
package/psplash: update for macaw

### PR DESCRIPTION
**Pull request by shubhamsharma-cpi**
This patch updates the hash of the psplash package after adding macaw logo image
and updates the build process to use the new image.

Signed-off-by: shubham sharma <shubham.sharma@chargepoint.com>

### CODE REVIEW

File: package/psplash/psplash.mk [hunk 1]
```diff
@@ -29,6 +29,12 @@ endef
 PSPLASH_POST_EXTRACT_HOOKS += PSPLASH_COPY_IMAGE
 endif
 
+ifeq ($(BR2_PACKAGE_EMB_MACAW_CHAPS),y)
+define PSPLASH_BUILD_CMDS
+    cp -a $(@D)/base-images/psplash-chargepoint-macaw.png $(@D)/base-images/psplash-chargepoint.png
```
Comment (chardin-cpi) [inline package/psplash/psplash.mk:34]: patch the COPY_IMAGE above and then make that configuration happen for macaw - ie. make this part

```
ifneq ($(PSPLASH_IMAGE),)
define PSPLASH_COPY_IMAGE
	cp $(@D)/base-images/$(PSPLASH_IMAGE) $(@D)/base-images/psplash-chargepoint.png
endef
```

Then add BR2_PACKAGE_PSPLASH_IMAGE=psplash-chargepoint-800x600.png into the macaw buildroot defconfig

Comment (shubhamsharma-cpi) [reply package/psplash/psplash.mk:34]: I added the PSPLASH_IMAGE definition in `mcb/buildroot_defconfig` as follows:
`BR2_PACKAGE_PSPLASH_IMAGE="psplash-chargepoint-macaw.png"`
However this definition is not visible in the `psplash.mk` file.

Comment (shubhamsharma-cpi) [reply package/psplash/psplash.mk:34]: I added the PSPLASH_IMAGE definition in `mcb/buildroot_defconfig` as follows:
`BR2_PACKAGE_PSPLASH_IMAGE="psplash-chargepoint-macaw.png"`
However this definition is not visible in the `psplash.mk` file.

**Comment by chardin-cpi** [comment]
obsoleted by #77

**Pull request closed without merging**